tests/test_comments.py

```diff
@@ -43,7 +43,7 @@ def test_incorrect_fixture(absolute_path):
     )
     stdout, _ = process.communicate()
 
-    assert stdout.count(b'E800') == 12 + int(PY36)
+    assert stdout.count(b'E800') == 6 + int(PY36)
```

Comment:
Can you please explain this change? Why do we have the error number decreased in half?

Reply:
Sure.

Well before `test_incorrect_fixture` was almost the same test as `test_incorrect_fixture_aggressive`. The differences were that the latter had `--aggressive` set und tested for `def ...` and `class ...` to be inside the output. 

But for me the first one failed because the amount of hits were too low and when I removed the `--aggressive` flag in the latter test it still succeeded. I looked it up in `eradicate` and saw that the lines with `def ...` and `class ...` should only be hits in the aggressive test but are counted for the target amount of hits in the non aggressive test.

So I lowered the amount in `test_incorrect_fixture` to the correct amount.